notebooks/Notebook2.md

```diff
@@ -18,17 +23,59 @@ exciting milestones, preliminary results, etc. Feel free to include images
 code, and any other resources that you think will help clearly convey your
 design process.
 
+I'm a little behind on work this week from a combination of ASP and EB,
+unfortunately... I basically worked on setting up the project a little more by
+creating the necessary html and javascript files. 
+
+I then fiddled around with the language that I wanted to create. While I had a
+vague idea last week, I tried to hammer out more of the details this week by
+actually creating a song in my language. First, I wrote out the sheet music
+using Noteflight. This took a weirdly long amount of time because it took me a
+while to figure out how long each beat actually was; I memorized the whole song
+a year ago, but I couldn't tell if something was a 16th note or 8th note. I was
+essentially just going off vibes the entire time. 
+
+Once I got what the sheet music would look like, I tried to figure out how I
+would write that in my language. One of the issues I quickly came across was the
+fact that there isn't really a specific sound associated with 16th notes. That
+is, a quarter note is typically represented by saying "don", eighth note is
+"do", but there isn't really something after that.
+
+To remedy this, I ended up adding a system where if I wrote "don[1/4]", that would mean
+I take the rhythm of the current note (don, a quarter note which is 1/4) and divide it by 4
+to get a sixteenth note (1/16). This is called a BeatMod (beat modifier). 
+
+I played around more with the language using Peggy.js (the parser)'s online editor so that 
```

Comment:
Peggy.js having an online editor is a really cool feature! I feel like more parsing packages (and just packages in general) should have something like that, as it lets you assess whether it will work for your purposes without having to commit to a download.